### PR TITLE
Skin api

### PR DESCRIPTION
**Pull request by Nano112**

### CODE REVIEW

File: app/Http/Controllers/Skin/SkinGetBody.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        } catch (\Exception $e) {
+            if ($e->getMessage() == "invalid_uuid") {
+                return response()->json([
+                    'errors' => [
+                        'uuid' => [
+                            'The uuid does not seem to be a valid UUID.'
+                        ]
+                    ]
+                ], 422);
+            }
```
Comment (lululombard) [inline app/Http/Controllers/Skin/SkinGetBody.php:34]: You probably want to handle other errors

File: app/Http/Controllers/Skin/SkinGetHead.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        } catch (\Exception $e) {
+            if ($e->getMessage() == "invalid_uuid") {
+                return response()->json([
+                    'errors' => [
+                        'uuid' => [
+                            'The uuid does not seem to be a valid UUID.'
+                        ]
+                    ]
+                ], 422);
+            }
```
Comment (lululombard) [inline app/Http/Controllers/Skin/SkinGetHead.php:33]: Same

**Review by lululombard — APPROVED** [review]
Looks great but you probably want to handle other exceptions if they can be thrown

File: app/Http/Controllers/Skin/SkinGetBody.php (hunk 1 above)
Comment (Nano112) [reply app/Http/Controllers/Skin/SkinGetBody.php:34]: https://github.com/redcraft-org/RedCraftWebsiteV3/blob/635e80ed206565488803d67148d85dcb53248419/app/Http/Requests/Skin/SkinBodyRequest.php

File: app/Http/Controllers/Skin/SkinGetHead.php (hunk 2 above)
Comment (Nano112) [reply app/Http/Controllers/Skin/SkinGetHead.php:33]: https://github.com/redcraft-org/RedCraftWebsiteV3/blob/635e80ed206565488803d67148d85dcb53248419/app/Http/Requests/Skin/SkinHeadRequest.php

**Review by lululombard — APPROVED** [review]

**Review by Cod3lta — APPROVED** [review]

**Pull request merged**